### PR TITLE
Demo Modal

### PR DESCRIPTION
**Pull request by mohitgusain**
#58 

1. Demo modal 
2. Including in header by the template

### CODE REVIEW

File: public/wp-content/themes/glific/header.php [hunk 1]
```diff
@@ -31,6 +31,8 @@
 		</a>
 		<div class="ml-auto d-xl-none">
 			<a class="bg-theme-primary font-heebo-bold px-10 py-4 fz-18 leading-27 text-white text-decoration-none ml-11 glific-button-border d-xl-none mr-5"
+				data-toggle="modal" \
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/header.php]: Typo, please self review once.
```suggestion
				data-toggle="modal"
```

File: public/wp-content/themes/glific/header.php [hunk 2]
```diff
@@ -56,8 +58,10 @@
 				'echo' => true,
 			));
 		?>
-		<a class="bg-theme-primary font-heebo-bold px-10 py-4 fz-18 leading-27 text-white text-decoration-none ml-11 glific-button-border d-none d-xl-block" href="#">Book a demo</a>
-
+		<a class="bg-theme-primary font-heebo-bold px-10 py-4 fz-18 leading-27 text-white text-decoration-none ml-11 glific-button-border d-none d-xl-block"
+			href="#"
+			data-toggle="modal"
+			data-target="#demoModal">Book a demo</a>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/header.php]:
```suggestion
		<a class="bg-theme-primary font-heebo-bold px-10 py-4 fz-18 leading-27 text-white text-decoration-none ml-11 glific-button-border d-none d-xl-block"
			href="#"
			data-toggle="modal"
			data-target="#demoModal">Book a demo
		</a>
```

File: public/wp-content/themes/glific/src/scss/style.scss [hunk 3]
```diff
@@ -16,83 +16,96 @@ Text Domain: coloredcow
...
+#glific_form_demo {
+	.last-col {
+		@extend .col-md-7;
+	}
+	.first-col {
+		@extend .col-md-5;
+	}
+	input[type="submit"] {
+		font-family: 'Heebo' !important;
+		font-weight: 700;
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/style.scss:107]: Any problem with `@extend` here? Can we use that?

File: public/wp-content/themes/glific/template-parts/book-demo-modal.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<div class="modal fade module-modal bd-example-modal-lg" id="demoModal" tabindex="-1" role="dialog" aria-labelledby="demoModalTitle" aria-hidden="true">
+	<div class="modal-dialog modal-dialog-centered modal-xl w-90p w-md-717 w-xl-805 max-w-full" role="document">
+		<div class="modal-content page-contact rounded-30 mx-auto">
+			<div class="contact-form bg-white p-8 rounded-30">
+				<?php
+				$demo_content = get_field('demo_content', 'option');
+				?>
+				<h3 class="text-theme-primary font-heebo-bold fz-28 fz-md-36 leading-40 mb-0 mb-8 mb-md-6 mb-xl-4"><?php echo $demo_content['text']; ?></h3>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/template-parts/book-demo-modal.php:8]: Can you put a check if `$demo_content` if empty?

**Review by Satendra-SR — APPROVED** [review]
@mohitgusain need minor changes and also have added some questions. Please fix and merge.

File: public/wp-content/themes/glific/src/scss/style.scss (hunk 3 above)
Comment (mohitgusain) [reply public/wp-content/themes/glific/src/scss/style.scss:107]: Yes , need to override the custom font classes by caldera form

**Pull request merged**